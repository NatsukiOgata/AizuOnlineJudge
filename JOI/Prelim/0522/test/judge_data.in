TJOILEJOISQMJOIZWZDJOIEEILCXUCDJKIHTMGBBHZYIJOIKGVUYWWJOIMZNRIAKRHUKVOWXUAJOIUGJOIBSUUJOIMJJQJOIOKDUDMPIDXJWJJOIGJOIDJOIRKDJOIEPYFNKTXTAFWIJOIAQWTAAQDSQVNGYETRMTDLFIJCKYOIFJNKADVWFCMFQREMSVEPGNZAPUIFTPCVAARIBPXEWOJOIZZEKLZOPIFHD
WLLFIIOITIOIJUOQUIGUKOIOIRFQKXRVLIOIMQBYMJTIOIKZNDIOICRYIOILTIOIZIOIAJUQNJGDPYZURIOINSJPKFCOQBCJFWAOWKXAUEDOIOIEJIOIBSCIOISIOIGXHEKVALKDLNDLFUDVJSYRURTVKWIOIANJWVIWFKJWMRFIOIDRKDCIJMOKBBOSDWNNUYIERMTWHIOIIILYQSEEZDITUZYFHPCQRIOIYIOIWJZCP
QAJXAIOILDZRSGSWUEVIRVLIOIIIYOQXDTKKUFSGLKCBZJOIZXBZOIQULJOIISLXOKIVKPFJTPBSRXZIOINJIRJOISKZCNOTYAXZRTHHXPEXDUZBUBNHKTUITJOIVOLYGKEROXOYVREKHPCJOIRNRJVHZDHIPGELHZVZZIOIXMFVYQVXJOIDAOZZRTLIOIZHIOITJUXHCGIUFTJOIAXIOIOJOIZDOWGDE
XJOIVPIOIFJOITPJLXCUBKLZJOIKHEIOIQXVCMLCCYPCKHQQCQHLXVWXHTKPYXZVXJOIVQZIOIPPYJBFOIXRGXNVRCFSLINOGIPXOYTJTIJIOIIXTQYVUYZANIYCRLLNIOINJOIKACIOIQTZHNBHREPOPBYJOIPDHRFLOXFPSAHGWRHSCLAETJOIDZIOIIOIOICKIOIAIOIOIZAIHCHFJOIWEYFNFPMALOBAZDZEUYVIHZPIOIWHZTVRMTVQOUOARVXZIOIOIBTGBSIOIGYZCFBJQYAADRIOIOIVBGPHEGUMATIOIIUXIYCZAMJGZLRXIOIOIQJSQTOQIHEPLSWJOIQAAIOIFFEPJOIQKMYXNIOIOIBPHSXUJOIIWLBBPAIOIQORWKJOIWJOIOHZHJOWBZIOIUSKAIOIVVPOUECVTZYABPIOINJOIXUIOIOSSLJCVVZQTNTOIOICIOIYJOIJJOILVKGLOTEGHSUJENGOQOGVWNPWQSTIOIPKNPAZBRYMNFBXTBFIOIZGGRWBXGDGAMNUTIDJOIVSSQIKRPAHIOIPHJOIGOSHAFMAUIOIXYYLMHKOBIOIJDUTIOIOIZJHQJOINJOILENIXVTFRWLOSBFZALWHIIGVPIOIKLNWLZXBTSWHJITDIOIRDRPMYERMYRGREOIOIKOIDTQEYIOIZMMXZFHZHHGTIOIIFOAVSGTIJOIRYBWZTWSJOIEHMIYHIPEYGGJLIOIVJOIRRTOUGAIOIAPVALXIVUNSJOITYUSGFSKJOIVMIOIROHCITOIOIPLAIBEYJOICPYELJOIFXJEIOIOIJMEIUALUISJBZIOICUHNHYBRIOIOIHIEYWAHJOIPSQMNDRIIOIKDIOIOIHYCIOIOICHUZUMLMHUWUBQLOFIOIGUOIOIRLXWALSOVHSTCPIUFWXEXJOIFCOTZOHIOIOIYDFVTBXLVPUYUGIOITVKIOIOIOIIFHOVJOIZEVIOIAIOIOIFPWGNQXIOIOJTFGYXGJOISJUIOIKBQKIEHXKPYREOEJOIXDQIIQQFQABIOITBIHWRIWCXYBEZUXUJOINPNJOIYZCIOIOIENQIAQIBOXRRIOIOIEOFIOIASTGKYDNKTWHMKIOIIENMKSYBZVFMCPBJOIVUIJOIWKWKOPSLJOIJJOIJFFAPEJOIWMMCFHXVEWQRHELYJOIWTAIOIHSFOLIOIVHHMIOIWICMLNMIHCNVISFQSPBRUVYIOIOFZDDTIOICJZOJOILWNKNYKIOISYNJOIXLZSQIVJTBIOIUVXINLTGDRTAHPPDWSKTMMBOEWAJOIDGZDQRQEDIOIIIJOIJIOIMEOJNUQEIOIFHUZEBQKJOIWHIOIRRSEGVNWOIOIVIOIETLGJOIPKJOITFWPJOIH
FJOIIGXFUPLIOIOZIOIYKIOIDJIJZULZMGJOIIFLCLZUNMVJOIUIIADZKSAYGTWXZLFMKYSNTJOIBEREJSYQVNVYCQOQZKBGJOIXFQVNDAXLSTFEDIOIKKCNQJOIEJOITJOIOIOIVHMWIOIZIOIHEQUZJRLGJOIOIXQMAOQQVIOIOJOIOILGMAEMBKNPJIOIYXLODBUCJOIGREVXBIOXLIOIVUCPOLMODLIOIOLPQJOIXRXTYJAAFNHGJOIXADWQCRMUOZAKIOIJAIOIQDSRLFBDHNTUTKJOILEMGTYBJJILRIOIRLMFJJBJOIBEYIOIPJVQVXTQYMPXMIOIJRYTIARKUSAIOIXVBDJEBTRGSSJOICDHIYRYKMRURJOIOIKFRMJOIFDUJOIGXSAEIMYTVIOIKGNCJOIJIOISZSTMHURKIOIXZIXEWYMIOIWVZOZVZXXYWYWJOIXRWZYZBAZWHRFLCBOIOIGTWPBAWARJKODFLSFVZWEJVQIOITNLKIOIWRDOGAPXIOIDPABPFSTJISAEFZNRDSQXSTOLWILEZOYSJIOIOIJOIOIIKIOIWRSLCOWHUADKJYMXZVIOIPIOWDTFXSSALKPOLJBNZXHGJOIQZXQETQOECZERBIQIOIIDVSRJOIUJWOBSIJOIIOIRQWXVHWLHJOIUBHPUORJEDTJOIHHMYXSMXNKGSMIOIPCAIOIVDWGNTQXGLJOIPBTMGXEXPIOINWECYEICLIDDYFXEBIOIBPYCRCLWRTYJUXTQGAHEVZITYKHBBERKBZUQYBYBULQQJJOIPEDNUKXFVSLYICTSWYIMLAHJIOIOIZIOITHJOITWYBNOTIFWZKIOIJRJOIOIRXXNFJOIOILBFDLFETIOIXVJOIRCQADNCGPYJOIBRLQAVMIYJOIHMIOILQHRCWDFIOIOIFIMJXRAIOICHGBVAUDKDCJOINPJOIOIIAJOILNTUBJOIZKGXLTJZNRRDKIZJOITRUIVMNIOIBTQCASIOIOIPFIOIMLEFUELXAIIOIUQGNLUXPTTGIOIOITEIOIVIOIIGKJOIRKIOIFPFFVPIOIIGMOKFYHIOIRIOIQILBOGCCJOILZSJIDCTIOIEBBBPDVGUHAAWRGHPRJQNSXPMJFOPJOIZIOIERWGJIOIOILGPUDGIOIXSIOIWATHSLKAEMNRTJSNAPRJOIOIHLNRRBWJOIXOTXJOIPKASWFIOISAKIOIYOCOHQTJOISEKYPNTTEZWBDRPRNPVAAFJOIVPMMSAATCQAJOIJOIOIRRENBXQKYVPTUSVTJOIPTIOIOIARRFDGRJOITAJOIRMOTDCWXJOIPNXVABJOIGFRZYJJOIGJOILLVIOITHRJOIIOIZHNBIOIWPCBGLNFUIOIKTJNORMRDLQRXARRLCQDJOIIOICPIOIAJGNLVIPANXWOKLUAUXJOIVYTMYKRIOINHFBCKGHPZBRZXVUMGLUAJOIHIOIOIPLCYOIOIBSQIOIOISBZIOIMGSMENNXZYQPSJIXIOIPQJIOINUZLDYXRIOIDJOIHVCJLCSNYTYXGXYEDKXKXJMCSGSBFZCIWGCIOIOHXCIMAMTWGJOIMSBZIOJPIOIZUFNEDLJOINMQEIOILJXIOILJMWKOBXEJOIAOBBFLKOJOIWIOIFPHCVBIOIQWLKTFFAOYZCATSJUJOIXCZRKCJOIIHKBJRMQBJOIBCCVWJCCIOIPQGVMDKJOILOGNEAZEDPCZIOIXYGRPFUFBRAIRTDJOIMTULFXSCIOIUEEQQCYMXWIJOIOIFYSXJXDAHKPPXLOOMIOIEIOIXSBIMJOIVCKUIOIXUJVAGORHTIOIAZIOIPOVGWWIUHHKZHMFNPGMHLVJIWORHJOISPJOIOITMVJAKRUJOIAWHACDJOIIOIKHZOVGXATCIOIMIOITMHJTWFZIZIPEIYTDPJOIAIOIIIOIOITJOIOIMDJOISVPISWKVTKJWXQQIOIOAFIOIAGBJOICOIKWAUJOZHJOIZKVVYBCTVMRWLWIFTSUDIOIBDFQJGFJOIXKTWSZIOIOIJEIUHYLVJOIOICMAJOIOIFJOIWJOIJMPIOIHVDFSNEOJOIINJOIFSYJOIFAIPEOIOHZMINEOLOHMIMMENYVKCCRPEMKIOIGYZJUKSDZBJIOIJTVAEJOIWBJOIXRPUHHXMZIJIOITKUIOINCRTISDKBEJTEIOIOBJOIKTTZGGDOVOEWMIOIKPXTJTAXGSTIOIWZJRSCSIWRMMAIOIXLBIJOIKZGOBJOIVINPJOIMRETEAHPIETJOIIOIAQSCSIOIFUJOIOIWHFMLALLDUDEYNRYBEIOIDSBEPNMIGLHKXGWGFQJMJOIBNYAIOINIPJOIZSAEVIOIDVGPQGIOITNTYJYIWIOIOIGRQTIOIZLHKWFLVQLXUIJOIDCSQIVPMBTATZGLJJDFXCLQXXTDXTKAJOIMKIXJOIIOIPFLOWNHRSRGPUHJOIIOISJOIUIJOIBUSIRQYXKIOIGEUDIOINIDURZCJXGASXJOIIOISEIUNCQJVLAUDVJOIJMNJOILSWCIOIOIYZJOIPMRIOIHJIEKIOIOIQZKNERSJOIOIOHZUPIALIOIJOIOIEZCXSWSKJFJEWIOIYVHNUIMWEVLJOICNWJOIBAPCGJOIOIWOVUOLVNYKWIUJWFJQJOIMNJKVBBWDJOIAZPDIAWGVWALYIOIJAWAIOIRRXKUHZJOIXCYVBWCAIOIRENHNSPUWFHWZSTDJGIRJOIXDWBCDEGPIWSEHCVIOIGQFQAFIOILQRQFDOHJOIHDIOIOJIVRRBUMXVBVDIOIKNPYIOIDTLMAQPHXFYPUQSWNDMUULFJOIIOIOJTBNZEIITVNTAJOIOIYBLIICXNTJOINPGJOIQLTDLVSXGIOIMFQCVIOIAJOIGMUQWYZHUKRDJOIRJOIHIOIRDIOIGIOIOISQTCJEJTTVCUSDVUZXZJOIOIHKAPNBMIOIOICRJOLAOIAFARDCIBBCVBZKKSMMFKIOISRSPTNLGDEEYWINJAAJOITGASMJOIOIXJBPFYCRBMHNCYMELGAIOIIAQVGBQOPQJOIXDZSIJOIIOIQGNGBHOFKLJSHPJOINLYOIPIOIJWCDQGOIOITIAFSXUZJOIFJOIWXCVJOIVJOIHUJOIBJOIDWLWIOIOIDJOIOIESFBJOFAIOIVPNQKSANLATHNIOIPUINBYNPYJOIIOIETGFWJOILYDZNHMGUJFDBDGUPPOIOIAQNXNKPHRUSIOIBZMDFGSHZBYJJPAJOITLGJUFJOIBIJIUDZJOIOINAEGJOIOINALSWFEJFAJOIRJOIDNKUTNFQZKWIOIOIWXIOIIXNBHOVGENDTRRITGHYRJOIPIOIBRNPPXKQARVFBSOIOIVBLZRAQQBZOMNACHHRWDXMZPCPBOJIOIOIJHKDXSLVANZZIOIZCLIOIKLZPJOIOIVXNOUZEHOIOIGGHIBFVUYIOIELOFUTBADJOIOIEPSTIOILDUPLWFIOIBOOJOIDIIOICWLWICPIJOINCEUCRBSDWJOIEZIOIOIZWCCGVIOIOIGUUFWJSVTCJOIRZUNAQPHMSNWMIOIFBRQFZTFNIOIJUNGSGRJOIGGCZZJOIABDRJOICLTWJKDRTJOIOIGCDFKJZRYSJOIOIFKIOIWIOIKHJKOKFDTUMXWIDFJOIXOONJEAPNJMOJOIOIOIDFXZHKDBBZVOSMOFCWXIOIFGWSTNEZHABJMVYCIUVQJOIJIOINJDAKWXJJOISCCLJDLTCCXZLQJHQXXCKZIOITSJOIAJOIYAIOILXLSGLYJPQUSFIOIKOIOIVIOIVHQEYSJOIZWVYLSOJVJOIFJOIOIVMCDZFBBPXIZJOIHIOIQAZRIIQIOIVSPISKPFQDTJOINJOIALRXBNIOIPOIOICGFCJRINJOIOIHNFCMBROVWBFAMUNIOIOIQIJOICIOIYEPGALWAIBZZODAPGIOIUUQTYQIOIIFZFJOIOITTAXHSKIFJOIQXCJJOIHKSIOIZANBBHXPNASEJOIAJOICEWVRIZHBZIOIGILMMTBJOIPSVXMCZJOIIOIRGWVHLATBLNWPONAFMNUJXRYCEKNYWJOIOIRJOIZLPNPNIYEVZZJOICVLTJOIOUVZMHSMIOIOIMHYYSVJOIHONWXIJOIMNIVIOIZIOIOIMFTLNLYYYCCJOISIOIYGJOIYEYZJOIDMFJOIDQGSKNNYEIOIUXLVXFRXPIASBKAQDRRRJOIJLPQYTFTDKMHZQRXEJIVJUYKRYCUJUQQUUKRTQJOIVTUHQAUIOIALRBIIDZJOIOINZEKIPICHIOIIUCGZBJRQNUXVJOILIOIUJOIULTQAMJOILKRVFIOIIYZTYAIOIPKTBOIIMZLBMKJPWUYOALDTMNDGWNGQUVXTXHSHVJOIPFEIOIVPKDBZKHHJOIVLGGPSPPKXDORCXEEJCMUVLEPAIHZNVJOILEJIFYCLGSLMEIAMRFDUGBNTLLJIJOIMCIOIOIAYNVPOSVRIOIOKMNYQJOIGJOIAIOIOIQSHRLZLVSYBJOIKMQAWYFESJOIQBABIEQLWXLSYGDKKNKNNPJYGJOISIOIYHWQIOIJBIOINRPHVGZJVAEZUIOIRKIOIOIOJOIIOICTJOIRSBVDLGTKWJMIOIOIPSDYCNIOIOISVGFNYHZJOIFYTDDIOIQOVVQBGHVJJUYVOTILIKQIOICSUVWFVNPWIBQLDLRMGZDZDVBHSJOIOIHZLYJOIXBIQUMIOILZLJJPCZOSHNZGKNZJOIZKDHMIKAYLOIGIOIOIXJOICQJOIIOIPHIXASAIEUYLNFSCSLTWTZYIUOZUZJDIOICJCTRUZPHPYIOIJHMOIOIFGGIQZIVCCJOILDPMHJILMLROJZZGIOIWLWSSSCREYILFJOILVEPNNCUEXNZOIOIEEGVVFKJOIKRTWXRWUJOIQHVMMMIOIUNQIOIOIWYHZKLJJOIOIBDBEQFNKMTEWKIOIUQZKXIOILMIOIIOIOIOXKFYPHTEIOIOIWSZDKLAQPWPYBOTNQAJRMNJOIPLFVGKJOIQZGZRAQOTIUZXCMMKJOIGJOIRPUJOIKIOIEMEWMIOIUZCYOSVCVQHZOJOIOITNQYKJDIOIKPIZZIJHICGWJHNEKOJSJOIBAWEZAJOIQIOIOIYIOIOIUTFIMFUIOIECJOIPWJOIGIJOIGDQZLRPNUIOIVPDIOIJHDCZAEUJOICJLCWGYKMCGJOIZOTXJOIKGMKAXIOIOIMUJJUZWZIXWIOIPIOICZFAIFMFNJOIUEXOZDNNITMORFCMIADTDYMWYXKJOIJKVIOIJOIOIPNSGYHFBMFJRIXWIOIGCBFHRKBHNIBJGCJOIDEIOIOFYIOIOINPLZIOIZMTGJOIPKAQDYZJOIQJOIDPULIOITSZMLVAVIOIUZHAKPBIOISXIOILLDRAOQWJOITGQUGIKLTUHSOSIIIBIOIOIKBSBGEXJOIYBXRBYQJIDJBMMVAGMOZWYRTIOIFJLAMIOIUIYPZCBFZIOIZGIJOIWRPVQQJOIOMZDJOIOIYCIYADGGRHAFNFSJOIIOISZQQSMVPIPWIOIEWYBIOIPJJOIHCOPQQDBYIOIQAZAEBELIOIOIBFXXWIOIXTZFXTJOIOBAZCOCJOIOIJRCECIOECRIJOIQNOEKYXDHCTLJOIYXJDWMOIOIGWGWKGLMFLEFCCRQUGMQVGJDGWDZMSEBXDOSBIZETAHEKWIDSHJIOIOIUPAHNGSPTVYBDKPVGZCJIKPQRUIOIIXRRLXVSYKKFURIOIOKXBLOLYGQIYBIHUKFURYQZOJOIUYBYMIOIOIWJOIXBIOIBZVUXIOIFBZBDPYPUPOWFLQYLRIOIBKJOINNCIOIOIOYHSLWXQKIMKSAJJKCJCIOIDWJIOIOSZPNAJERRGMIOIWSXETETZXZICYAPYSYTQCNGJOIHGRUZIOIIXROCJKPENNHLEIOITCGEPAESBDNRFSVJOICRQJOINQAWERIBQWPPHILOOIOITLTIIOILKRBUWEFOMWLJOIGFGIOIEMDIQWJOIBJOIOIJYCJOIAPATJOIPPNXAAPOHLOEDGCKIOIOIDZOJRBIOIOIOHWPGVUIOIINKNNEMQTIOIOIVVMJOIOIRHMCFGVUQWNIOIOWKBFRHNLIOIQRBQVCDIOILLGPQLFWLFMMEGDUSGIOIOIDHEIMTHIOIOIJIOIYFLIKIOIOHIOIOCAWSHHROCMHINRTQJJOIFJOIWSMXCGIOIPNNCJOILDMLNMIVVIOIIXJOIOIKFAEQNITGOGOLEXJOIHAQUKSVUWIIOIGKFIOIPLIRWTMXMYQIJYYJMWIREVASDMGWKXMTEAFFAKLPOIOIIOIOIWZQDMZQFAMSFTIBTIOIQTJOIITYLFFQODTQPLWOJOIPINYIOMKHHMTAIOIOICITISIJOIMHXRUQIOIRWQVOXSEJOIQMZXNIOILDHWIZMOAPNBBJOINDCYDIODNEFPFGHJOIVWZUSOTNXIKIOIDLFQUSIOISEGXRFAFIOIATOEFJOIWOJTVJOIOIUHGOSHOLOOYQAZRIOIXQRSNXQTYYJOIJFVHSNZXQIOIOIEGIOIONCPQUIOIJBHMRWTZTCMVBJOIFQGZMIOIUKXCONNBPNEWFXJFJOIJOIOIRSTTATRNRFIOIOIDGZJNZXYJIOIOIZFGTVZBXNCEEXVBUNIOIXNPYWJOIAIOIXUIJOIOIPCAJOIOICSDJOIPKLPBIIOIWJOIBXFDJOIEUXYMFYBWFYLAVCGKIOIYFXGBQCDYXOGOIOIKXDZQMEIOIZRQEVJJOIOJOIBNRDFVTUNQLSSHKTXIOIFRVBLFJOIXCEQYNZDIHJOIAJOIIOINGIOITKTLOULDVPAUCRYTGQKFHJNZDLURSWEGJOILYJOIRVSCGQLBPBWJOIZGJOIZHTNILEWGMSHJEPCBYNPVFINIOIWSJOIJKVXHQJOICIOIZQIIOIDDMKQIOILYIEIZWYIUCIXJY
