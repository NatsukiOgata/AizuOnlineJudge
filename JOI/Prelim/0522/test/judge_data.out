16
0
0
19
8
7
48
90
297
418
